src/components/forms/Input.jsx

```diff
@@ -8,20 +8,22 @@ import { MAIN_DARK_BLUE, MAIN_GREEN } from '../constants'
 import BorderedBox from '../BorderedBox'
 
 function Input ({ placeholder, value, name, borderColor, errorMessage, onChange, disabled, beforeIcon, afterIcon, focused, placeholderApart }) {
-  let className = styles.inputContainer + ' ' + commonStyles[`bordered--${borderColor}`] + ' ' + commonStyles[`text--${borderColor}`] + ' ' + commonStyles.padded
+  let inputClassName = `${commonStyles.fullWidth} ${styles.input} `
```

Comment:
There is probably a better way to manipulate class names than string concatenation, but won't block for that